zerver/migrations/0201_zoom_video_chat.py

```diff
@@ -22,9 +22,4 @@ class Migration(migrations.Migration):
             name='zoom_api_secret',
             field=models.TextField(default=''),
         ),
-        migrations.AddField(
```

Comment:
For most intents and purposes you can't modify old migrations. To remove the field you'll want to create a new migration that removes it.

Reply:
Which `manage.py makemigrations` will do for you after you remove this change.